### PR TITLE
ADF-3067: Retrofit Facebook Audience Network for Advanced Bidding

### PR DESCRIPTION
**Pull request by kdun**
Merge for SDK 5.0

### CODE REVIEW

File: FacebookAudienceNetwork/FacebookAdvancedBidder.m [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+#import "FacebookAdvancedBidder.h"
+#import <FBAudienceNetwork/FBAudienceNetwork.h>
+
+@implementation FacebookAdvancedBidder
+
++ (void)initialize {
+    if (self == [FacebookAdvancedBidder self]) {
+        // Initialize an adview to trigger [FBAdUtility initializeAudienceNetwork]
+        FBAdView * dumbHack = [[FBAdView alloc] initWithPlacementID:@"" adSize:kFBAdSize320x50 rootViewController:[UIViewController new]];
```
Comment (rsatrasala) [inline FacebookAudienceNetwork/FacebookAdvancedBidder.m]: Do we really want to leave dumhack variable?

Comment (kdun) [reply FacebookAudienceNetwork/FacebookAdvancedBidder.m]: it can be renamed, but fundamentally FB never fixed the issue as they promised

Comment (rsatrasala) [reply FacebookAudienceNetwork/FacebookAdvancedBidder.m]: Can I call it initHackforFacebook?

Comment (kdun) [reply FacebookAudienceNetwork/FacebookAdvancedBidder.m]: sure

**Pull request merged**